logstash-core/lib/logstash/modules/kibana_config.rb

```diff
@@ -28,26 +28,22 @@ def dashboards
     end
   end
 
-  def kibana_index_hacks
-    # Copied from libbeat/dashboards/importer.go
-    # CreateKibanaIndex creates the kibana index if it doesn't exists and sets
-    # some index properties which are needed as a workaround for:
-    # https://github.com/elastic/beats-dashboards/issues/94
-    # with kibana 5.4.0 this hack failed to be applied.
-    ha = '{"settings": {"index":{"mapping":{"single_type": false}}}}'
-    hb = '{"search": {"properties": {"hits": {"type": "integer"}, "version": {"type": "integer"}}}}'
+  def index_pattern
+    pattern_name = "#{@name}-*"
```

Comment:
I take that back. It should default to `logstash-*`. It shouldn't depend on module name.